src/modules/NeighborInfoModule.cpp

```diff
@@ -87,7 +88,8 @@ void NeighborInfoModule::cleanUpNeighbors()
     NodeNum my_node_id = nodeDB->getNodeNum();
     for (auto it = neighbors.rbegin(); it != neighbors.rend();) {
         // We will remove a neighbor if we haven't heard from them in twice the broadcast interval
-        if ((now - it->last_rx_time > it->node_broadcast_interval_secs * 2) && (it->node_id != my_node_id)) {
```

Comment:
check that getTime() is equivalent to millis()

Reply:
+ remove now variable.

Reply:
getTime:" * Returns the current time in seconds since the Unix epoch (January 1, 1970)."